Tests/UnitTests/RequestBuilderTests.swift

```diff
@@ -43,6 +45,15 @@ class RequestBuilderTests: XCTestCase {
         XCTAssertEqual("A", requestPayload?.meta?.konductorConfig?.streaming?.recordSeparator)
         XCTAssertEqual("B", requestPayload?.meta?.konductorConfig?.streaming?.lineFeed)
         XCTAssertTrue(requestPayload?.meta?.konductorConfig?.streaming?.enabled ?? false)
+        XCTAssertEqual("datastreamID", requestPayload?.meta?.sdkConfig?.datastream?.original)
+
+        guard let requestConfigOverrideMap = requestPayload?.meta?.configOverrides,
+              let test = requestConfigOverrideMap["test"]?.dictionaryValue as? [String: Any],
+              let value = test["key"] as? String else {
+            XCTFail("Invalid config overrides payload.")
```

Comment:
nit: as there is multiple places where this could fail, include the `requestPayload` in the fail message.